src/frontend/Komponenter/Behandling/Formkrav/utils.ts

```diff
@@ -74,7 +74,8 @@ export const erVedtakFraFagsystemet = (valgtElement: string) => {
         valgtElement === PåklagetVedtakstype.UTEN_VEDTAK ||
         valgtElement === PåklagetVedtakstype.IKKE_VALGT ||
         valgtElement === PåklagetVedtakstype.INFOTRYGD_TILBAKEKREVING ||
-        valgtElement === PåklagetVedtakstype.UTESTENGELSE
+        valgtElement === PåklagetVedtakstype.UTESTENGELSE ||
+        valgtElement === PåklagetVedtakstype.INFOTRYGD_ORDINÆRT_VEDTAK
```

Comment:
Ikke helt koblet akkurat til denne PR men lurer på om denne metoden egentligen bare burde kunne sjekke for at `valgtElement === PåklagetVedtakstype.VEDTAK` 😬